cypress-tests/cypress/support/commands.js

```diff
@@ -1466,85 +1545,87 @@ Cypress.Commands.add(
     }).then((response) => {
       logRequestId(response.headers["x-request-id"]);
       expect(response.headers["content-type"]).to.include("application/json");
-      if (response.status === 200) {
-        globalState.set("paymentID", paymentIntentID);
-        globalState.set("connectorId", response.body.connector);
-        expect(response.body.connector, "connector").to.equal(
-          globalState.get("connectorId")
-        );
-        expect(paymentIntentID, "payment_id").to.equal(
-          response.body.payment_id
-        );
-        expect(response.body.payment_method_data, "payment_method_data").to.not
-          .be.empty;
-        expect(merchantConnectorId, "connector_id").to.equal(
-          response.body.merchant_connector_id
-        );
-        expect(response.body.customer, "customer").to.not.be.empty;
-        expect(response.body.billing, "billing_address").to.not.be.empty;
-        expect(response.body.profile_id, "profile_id").to.equal(profileId).and
-          .to.not.be.null;
-
-        validateErrorMessage(response, resData);
-
-        if (response.body.capture_method === "automatic") {
-          if (response.body.authentication_type === "three_ds") {
-            expect(response.body)
-              .to.have.property("next_action")
-              .to.have.property("redirect_to_url");
-            globalState.set(
-              "nextActionUrl",
-              response.body.next_action.redirect_to_url
-            );
-            for (const key in resData.body) {
-              expect(resData.body[key], [key]).to.deep.equal(
-                response.body[key]
+      cy.wrap(response).then(() => {
```

Comment:
include content type check within wrap() and:

```suggestion

      cy.wrap(response).then(() => {
```